app/controllers/users_controller.rb

```diff
@@ -15,7 +15,121 @@ def show
 	    @time = current_user.time
 	    @cuisine = current_user.cuisine
 		@calories = self.class.calc_calories(@gender, @weight, @height, @age, @exercise, @goal)
-		Recipe.find_in_api
```

Comment:
need to replace this line back to real api call after the api call method is done, but can keep the mock data for later use as well